recipes/openmesh/bld.bat

```diff
@@ -0,0 +1,24 @@
+rem  @echo off
+
+mkdir build_win
+cd build_win
+
+set CMAKE_CONFIG="Release"
+
+cmake -LAH -G"NMake Makefiles"                               ^
```

Comment:
You may want to add `-LAH` if you see issues as this prints all the CMAKE variables and makes debugging easier.